NewHaven/src/Player.h

```diff
@@ -17,13 +17,13 @@ struct Player
     // cannot implement yet. Requires the game_controller / main-loop / logic controller to be a singleton with references to the harvest tile
     void drawHarvestTile(HarvestTile& tile);
     // im assuming this prints out the available resources;
-    ResourceScore resourceTracker();
+    ResourceTracker resourceTracker();
     bool buildVillage();
```

Comment:
`ResourceTracker getResourceTracker() const;`
Is more readable, I had to look at the implementation to understand what the function did, while you could just make it clear in the definition.

Reply:
resourceTracker is the name the teacher wants in the assignment unfortunately. I share your opinion on its readability.